site/research/transaction-extensions/index.html

```diff
@@ -434,7 +434,7 @@ <h2 id="references"><strong><i class="fa fa-book"></i> References</strong><a cla
         <footer class="col-md-12">
             <hr>
             
-                <center>dcrdocs v0.0.1. Decred Project 2016.</center>
+                <center>If you wish to improve this site, please <a href="https://github.com/decred/dcrdocs/issues/new"><i class="fa fa-github"></i> open an issue</a> or <a href="https://github.com/decred/dcrdocs/compare"><i class="fa fa-github"></i> send a pull request</a>.<br>dcrdocs v0.0.1. Decred Project 2016.</center>
```

Comment:
Should it maybe be version 0.0.2 (since this in no longer the initial version of the docs)?